plugins/auditlib/lib_auditlib.ecllib

```diff
@@ -0,0 +1,4 @@
+export AuditLib := SERVICE
```

Comment:
I think this needs a plugin attribute - so eclcc knows which .so to link in.

Reply:
Resolved